library/custom_template/custom_template.php

```diff
@@ -51,6 +51,14 @@ function listitemCode($strDisp, $strInsert)
 ?>
 <html>
 <head>
+<style>
+    .draggable {
+        cursor: pointer !important
+    }
+    .is-dragging {
+        cursor: move !important
+    }
+</style>
 <?php Header::setupHeader(['common', 'opener', 'jquery-ui', 'select2', 'ckeditor']); ?>
```

Comment:
hi @im-Amitto , does the above `jquery-ui` have to go as well?

Reply:
Thanks for the review. Forgot to remove it after removing the sortable function which was using it.